src/operations/hulls/hullChain.test.js

```diff
@@ -30,11 +30,15 @@ test('hullChain (three, geom2)', t => {
   let obs = hullChain(geometry1, geometry2, geometry3)
   let pts = geom2.toPoints(obs)
 
-  t.is(pts.length, 10)
+  // the sides change based on the bestplane chosen in trees/Node.js
+  // t.is(pts.length, 10)
+  t.is(pts.length, 11)
```

Comment:
same question as a bit above: how come the amount of points changed ?

Reply:
Because the union of the hulls produces slightly different results, due to the choice of bestplane.
I checked thoroughly and found that results were valid for each.